backend/src/main/resources/liquibase/db-changelog-2.0.xml

```diff
@@ -979,4 +979,24 @@
         <tagDatabase tag="db_version_2.34"/>
     </changeSet>
 
+    <changeSet id="marked-existing-artifact-uploaded-true" author="anandJ">
+        <preConditions onFail="MARK_RAN">
+            <columnExists tableName="artifact" columnName="upload_completed"/>
+        </preConditions>
+        <update tableName="artifact">
+            <column name="upload_completed" valueBoolean="true" />
+            <where>field_type="artifacts" AND upload_completed = 0</where>
+        </update>
+        <rollback>
+            <update tableName="artifact">
+                <column name="upload_completed" valueBoolean="false" />
+                <where>field_type="artifacts" AND upload_completed = 1</where>
+            </update>
```

Comment:
This rollback is not valid. This is an information losing change, so we can't rollback from it. Let's remove to make that clearer.

Reply:
Ok, got it. removed rollback tag now.